orb.yaml

```diff
@@ -59,8 +72,8 @@ commands:
             source $BASH_ENV
             echo -e "machine api.github.com login x-access-token password $GH_TOKEN" >> ~/.netrc
             rm .circleci/github.key.pem
-  generate-nais-deployment:
-    description: "Generate NAIS Github deployment"
+  deploy-with-gai:
```

Comment:
I stedet for with-gai og with-pat hadde det kanskje vært mer beskrivende med with-gh-app og with-personal-token